src/components/picker/NativePicker.js

```diff
@@ -1,14 +1,12 @@
 import _ from 'lodash';
-import React from 'react';
-import {BaseComponent} from '../../commons';
+import React, {Component} from 'react';
 import TextField from '../textField';
 import PickerDialog from './PickerDialog';
 import TouchableOpacity from '../touchableOpacity';
 import {Colors} from '../../style';
 import {WheelPicker} from '../../incubator';
 
-class NativePicker extends BaseComponent {
-  static displayName = 'IGNORE';
```

Comment:
Should we add the displayName?

Reply:
Fixed. 
It helps when errors prints callstack